### ISSUE TITLE
Hey I made a thing...

### ISSUE THREAD

**Issue by MichalBryxi** [owntracks/booklet#84]
## The story

Once upon a time I used to use [Google Latitude](https://en.wikipedia.org/wiki/Google_Latitude) to share real time location with friends. Then Google killed it and years later introduced alternative in [Google Maps](https://support.google.com/maps/answer/7326816?hl=en&co=GENIE.Platform%3DDesktop). Then a bit while later I started looking into a zero touch, (near) real time location sharing of my location with my partner where I would own the data and I could also control their publication. All that for security purposes. After vetting out a _lot_ of shady apps on the Play Stores I concluded that there is no solution I would be happy using.

So what do you do when you _can_ build an app? Yup, you build an app and regret it later. The app is storage agnostic and for simplicity the first provider is built on [firebase](https://en.wikipedia.org/wiki/Firebase) and the client for collecting and publishing data was built as [Automate script](https://llamalab.com/automate/). All was well and nice, except the Automate script was clunky, not easily configurable and buggy in edge cases.

Then someone on Fediverse [pointed me to Owntracks](https://veganism.social/deck/@recollir@mountains.social/112585353180619809) and I thought: "heh, maybe that would solve my problems". I looked into the [Quicksetup](https://owntracks.org/booklet/guide/quicksetup/) and frankly got scared of the complexity of the "quick" way of setting this up. So I ran an experiment to see what will happen if I point [OwnTracks Android client](https://play.google.com/store/apps/details?id=org.owntracks.android&hl=en_US) against my firebase provider. And it worked! So I moved the Device part of my app to [use OwnTracks](https://github.com/vast-ch/the-mountains-are-calling/pull/51/files) and called it a day. Obviously trying to give enough exclamation marks around "this is not something that should be considered secure, unhackable or private". And by no means it has fraction of the functionality of OwnTracks.

But(!) it does have _significantly_ [shorter quickstart](https://github.com/vast-ch/the-mountains-are-calling?tab=readme-ov-file#quickstart). And that's what I'm using now. And since you [called out for letting you know](https://github.com/owntracks/booklet/blob/e5ad20448dcf5161da911f3d5f2569725accb1f8/docs/guide/clients.md?plain=1#L82), I'm letting you know.

## tl;dr

I made an app called [The Mountains Are Calling](https://github.com/vast-ch/the-mountains-are-calling?tab=readme-ov-file) for quick & easy & reliable location sharing for security purposes. Turns out OwnTracks Android/iOS client is a perfect part of that puzzle to gather the location data and store it in my quick-setup provider.

**Comment by MichalBryxi** [comment]
To maintainers: Maybe you want to mention my project in your handbook. Maybe it's too quirky & actually might violate your idea of privacy (as per-design limited functionality). So maybe you just want to close the issue. Either way, it's ok for me.
I just wanted to let you know and also thank you for your client <3

**Comment by jpmens** [comment]
Thanks for your report. I've [mentioned this](https://owntracks.org/booklet/guide/clients/#related) in the Booklet.
